### DISCUSSION TITLE
Key encryption

### DISCUSSION THREAD

**Discussion by DamianGlowala** [digitalfortress-tech/localstorage-slim#18] [Ideas]
Rather than being able to encrypt only the value, is it feasible to add the encryption for the keys?

**Comment by niketpathak** [comment]
Hello, Thank you for your suggestion, although, I do not see a real-world use case for encrypting the key. If this is something that other users desire, kindly up-vote this idea so we can give it a shot (with examples of where this might be needed). 
Thank you
